platformcomponents/desktop/tag.json

```diff
@@ -29,6 +29,23 @@
         "border": "@theme-outline-primary-normal"
       }
     },
+    "common": {
```

Comment:
what is the difference between the common tag and that overlay tag? The overlay tag has a white background, black text, and gray border. And you said you did not need hover, pressed, disabled states.

Reply:
There is no difference, did push commit [Removed common style from tag component](https://github.com/momentum-design/tokens/pull/212/commits/a3dbf6c284e3a50b797166f44d7d3d76744324e8)
Changing this conversation to resolved.